.circleci/config.yml

```diff
@@ -20,7 +20,7 @@ workflows:
           requires:
             - yarn/workflow-queue
       - yarn/jest:
-          args: --runInBand
+          args: --runInBand --no-cache -u
```

Comment:
We'll need to immediately follow up with a PR that removes these flags. Couldn't figure out how to bust circle's cache and ran into that snapshot update issue.